client/account-recovery/reset-password/alternate-access-form/index.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+import { identity, noop } from 'lodash';
+import classnames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import support from 'lib/url/support';
+import Card from 'components/card';
+import Button from 'components/button';
+import FormLabel from 'components/forms/form-label';
+import FormInput from 'components/forms/form-text-input';
+
+export class AlternateAccessFormComponent extends Component {
+	static defaultProps = {
+		translate: identity,
+		label: '',
+		description: '',
+		onSkip: null,
+		onContinue: noop,
+	};
+
+	state = {
+		isSubmitting: false,
+		fieldValue: '',
+	}
+
+	submitForm = () => {
+		this.setState( { isSubmitting: true }, () => {
+			this.props.onContinue( this.state.fieldValue );
```

Comment:
I figure at the point were the components state definitely has `isSubmitting===true` that means the form is locked and no further updates can be made to `fieldValue`. Its probably being a little paranoid but I don't think it hurts